Viprinet customer router management domains

SimonKissel opened the pull request:
Viprinet is a manufacturer of VPN Routers. SERIALNUMBER.router.management (.management is a new public TLD) are used to access our routers. In future, we also wish to support Let's encrypt, which requires this domain to be on the public suffix list. Subdomains of the vipri.net domain are internally used for update servers and the likes, and therefore also should be on the public suffic list.

SimonKissel commented:
Both domains are btw owned by Viprinet Europe GmbH, with myself listed as ADMIN-C, so verification should be easy.

sleevi commented:
Sorry for the delay @SimonKissel 

Can you please update this patch according to the [Guidelines]( https://github.com/publicsuffix/list/wiki/Guidelines)

In particular:
* Rebase to master
* Ensure that your header includes submitter information
* Indicate the means to authenticate this request (preferably, DNS TXT record)

In addition:
* Your description about "vipri.net" subdomains being used as update servers does not necessarily mean they need to appear on the PSL. You should only add if your subdomains are operated by independent entities. Is this case? Please review the Guidelines document to ensure you understand the affects this may have on your domain operation and analytics.

SimonKissel commented:
Hi Sleevi,

thanks for proceeding on my request. You are correct, vipri.net doesn't really have to be on the PSL, I will remove that one.

SimonKissel commented:
I hope to correctly have rebased to master. I have also removed the vipri.net domain, and added submitter info.

I'd like to authenticate to DNS TXT:

dig TXT _psl.router.management @ns1.viprinet.com
_psl.router.management. 21600   IN      TXT     "https://github.com/publicsuffix/list/pull/96"

sleevi commented:
Verified via TXT

sleevi commented:
Please make sure the header conforms to the [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines), specifically, remove the date.

SimonKissel commented:
Date removed. OK now?

SimonKissel commented:
Hi Sleevi,

I might have problems to understand how this stuff works, but why doesn't the domain show up in public_suffix_list.dat (or any other pull request higher than #64, for that matter)?
